doc/examples/mnist_mlp/mnist_loader.hpp

```diff
@@ -42,18 +42,18 @@ class MNistLoader
         return fread(loc, sizeof(T), n, m_file);
     }
 
-    std::uint32_t get_items() { return m_items; }
+    uint32_t get_items() { return m_items; }
 protected:
     std::string m_filename;
-    FILE* m_file{nullptr};
-    std::uint32_t m_magic;
-    std::uint32_t m_items;
+    FILE* m_file;
+    uint32_t m_magic;
```

Comment:
These should be coming from `<cstdint>` and be `std` according to https://stackoverflow.com/questions/14883896/why-is-stduint32-t-different-from-uint32-t
Do you have a source that says otherwise?

Reply:
I think `std::uint32_t` is an *alias* of `uint32_t` *IF* both come from `cstdint`

Reply:
This is from <cstdint>. It is as @Krovatkin says, `std::uint32_t` is just an alias to `uint32_t`
```
namespace std
{
  using ::int8_t;
  using ::int16_t;
  using ::int32_t;
  using ::int64_t;

  using ::int_fast8_t;
  using ::int_fast16_t;
  using ::int_fast32_t;
  using ::int_fast64_t;

  using ::int_least8_t;
  using ::int_least16_t;
  using ::int_least32_t;
  using ::int_least64_t;

  using ::intmax_t;
  using ::intptr_t;

  using ::uint8_t;
  using ::uint16_t;
  using ::uint32_t;
  using ::uint64_t;

  using ::uint_fast8_t;
  using ::uint_fast16_t;
  using ::uint_fast32_t;
  using ::uint_fast64_t;

  using ::uint_least8_t;
  using ::uint_least16_t;
  using ::uint_least32_t;
  using ::uint_least64_t;

  using ::uintmax_t;
  using ::uintptr_t;
} // namespace std
```